docs/developers/contracts/technical-references/tick-ratio.md

```diff
@@ -1,100 +1,96 @@
+* On the _bids_ side, however, the price is the reciprocal of the $wants/gives$ ratio, i.e. price = $gives/wants$. This is because a bid wants base tokens and offers quote tokens and thus the ratio $wants/gives$ is in `base/quote`.
 
-### 2. Tick tree
 
-As previously said, offers are stored in bins, which are sorted in a "tick tree". Thanks to this tree structure, **offer operations (insert, update, and retract) take constant gas** (the height of the tree is fixed).
+:::info Example
+On a WETH/DAI market:
+* The price would be expressed in DAI per WETH (DAI/WETH)
+    * Eg. 2,224 DAI/WETH
+* On the asks side WETH-DAI (selling WETH or buying DAI), the ratio equals the price and has unit DAI/WETH
+    * Eg. 2,224 DAI/WETH
+* On the bids side DAI-WETH (buying WETH or selling DAI), the ratio equals 1/price and has unit WETH/DAI.
+    * Eg. 0.0004496 WETH/DAI
+:::
 
-### 3. Tick bins
 
-Below the bottom of the tree are tick bins - they are laid in sequence. In the context of an [offer list](./taking-and-making-offers/offer-list.md), each tick bin has an associated tick (and a tick determines a price).
+:::warning Beware decimals!
+Note that ratio and price are in raw numbers, ie, disregarding token decimals. Care must be taken when converting raw prices and ratios to their human readable counterparts with decimals.
+:::
 
-:::danger Important
-All offers in a bin have the same tick. During a [market order](./taking-and-making-offers/taker-order/README.md), **offers in a tick bin are executed in order**, from the first to the last. Inserted offers are **always appended at the end of a tick bin**.
+:::info Remember
+We often casually use the word ’price’ to refer to the ratio between the amount promised by an offer and the amount it requests. In the code however, we use the generic word ’ratio’ to avoid confusion with the normal notion of market price expressed in ’quote’ tokens per ‘base’ token.
 :::
 
 
-### 4. `tickSpacing`
+## Converting a price to a tick
 
-As previously said, each tick points to a single tick bin, where all offers with an identical price are stored. We use a `tickSpacing` value to link the tick bins (i.e. the ticks, i.e. the prices) of an offer list together. If a bin has tick `t`, the following bin has tick `t+tickSpacing`.
+Since the price space is discretized, not all prices can be represented exactly. Converting a price to a tick may therefore require rounding and one should take care to round appropriately; We'll discuss this in detail below.
 
-:::info Note
-* A larger `tickSpacing` means the jumps in tick (i.e. price) are bigger.
-* A smaller `tickSpacing` means the jumps in tick are narrower.
-    * Example: a high-volatility market would require a larger `tickSpacing` compared to a more stable one.
-:::
+First, calculate an exact but possibly non-representable `tick'` which may have a real component:
 
-### 5. Our approach to ticks
+$$
+tick`_{side \in \{asks, bids\}}(price) = \begin{cases}
```

Comment:
I would use ratio here, see next comment.

Reply:
Addressed in #203.

I added a formula for ratios as well, as I think people may want to go both `ratio -> tick` and `price -> tick`.